content/accessibility/announcements.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Announcements
+---
+
+## Overview
+
+The guidelines below help answer the question “Should I ensure assistive tools output a message in _<this_situation>_?”
```

Comment:
```suggestion
These guidelines will help you determine when assistive tools should output a message.
```

Reply:
trying to make it fit with other overviews and remove the "question" format. let me know if I misunderstood the meaning

Reply:
Your suggested wording is clearer; thanks @yaili! 🙌